From 673d8502495b71003d252c861d54cda5c67e8dd3 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Mon, 21 Dec 2015 10:50:48 +0800
Subject: [PATCH 0786/5242] MLK-12042 ARM: dts: imx6sl.dtsi: add mmc aliase

commit  2274fc892b4f34a8a04154a7f2eb4cf38b770945 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch add mmc aliase support, to make every sd slot index
fixed.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sl.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl.dtsi b/arch/arm/boot/dts/imx6sl.dtsi
index 85e0565..c10a711 100644
--- a/arch/arm/boot/dts/imx6sl.dtsi
+++ b/arch/arm/boot/dts/imx6sl.dtsi
@@ -25,6 +25,10 @@
 		gpio2 = &gpio3;
 		gpio3 = &gpio4;
 		gpio4 = &gpio5;
+		mmc0 = &usdhc1;
+		mmc1 = &usdhc2;
+		mmc2 = &usdhc3;
+		mmc3 = &usdhc4;
 		serial0 = &uart1;
 		serial1 = &uart2;
 		serial2 = &uart3;
-- 
1.7.9.5

